### DISCUSSION TITLE
DECISION: mobile version, multi device sync.

### DISCUSSION THREAD

**Discussion by alexaunter** [galactic3/ft-lockup-contract#36] [Q&A] [answered]
as it was decided with @frol on 2022-02-14,
we DONT need to sync data between devices, we don't require users to log in to see any of the publicly available data
the UI app works like explorer where everyone is able to see all data for any lockup in user friendly format.
When user wants to perform some write action, then transaction data is created and user is able to sign it in any app.
Sounds like the app doesn't really need the ability to log in. Did I get it right, @frol?

**Comment by frol** [comment, marked as answer]
> Sounds like the app doesn't really need the ability to log in. Did I get it right?

Correct. We discussed that we can get away with the read-only UI most of the time, and we will only need to sign individual actions.
